modules/autocomplete/test/spec.coffee

```diff
@@ -4,3 +4,21 @@ describe 'autocomplete', ->
     hx.userFacingText('autoComplete','noResultsFound').should.equal('No results found')
     hx.userFacingText('autoComplete','otherResults').should.equal('Other Results')
     hx.userFacingText('autoComplete','pleaseEnterMinCharacters').should.equal('Please enter $minLength or more characters')
+
+
+  it 'should sort items correctly when using objects', ->
+    itemsObjects = hx.range(100).map (i) -> id: i
+
+    ac = new hx.AutoComplete hx.detached('div').node(), itemsObjects
+
+    ac.show()
+    ac._.menu.items().map(({id}) -> id).should.eql(hx.range(100))
+
+  it 'should sort disabled items correctly', ->
+    itemsObjects = hx.range(100).map (i) -> id: i, disabled: i % 2 == 0
```

Comment:
Another coffeescript syntax ambiguity.. Is this being mapped to an object? Can it be made more explicit with curly brackets?